exercises/scripts/Setup-Databases.sql

```diff
@@ -3,144 +3,176 @@ GO
 
 -- Exercise-01-before
 
-IF  NOT EXISTS (SELECT name FROM master.dbo.sysdatabases WHERE name = N'Exercise-01-before-sales')
-CREATE DATABASE [Exercise-01-before-sales]
+:setvar DatebaseName "Exercise-01-before-customers"
```

Comment:
Typo: should be "DatabaseName" (and all other instances).

Reply:
Changed